alacritty_macos.yml

```diff
@@ -200,5 +200,10 @@ mouse_bindings:
 
 # Shell
 #
-# You can set this to a path to your favorite shell, e.g. /bin/fish
-shell:
```

Comment:
Sure, what is the correct implementation of this though?

To me it would be more logical to have it commented out. Having an empty string (or null or whatever) as the default seems wrong to me.